src/reconcile/rec_write.c

```diff
@@ -1871,21 +1857,11 @@ __rec_split_write(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REC_CHUNK *chunk
           session, btree->maxleafpage, compressed_size, last_block, &btree->maxleafpage_precomp);
 
 copy_image:
-#ifdef HAVE_DIAGNOSTIC
-    /*
-     * The I/O routines verify all disk images we write, but there are paths in reconciliation that
-     * don't do I/O. Verify those images, too.
-     */
-    WT_ASSERT(session, verify_image == false ||
-        __wt_verify_dsk_image(
-          session, "[reconcile-image]", chunk->image.data, 0, &multi->addr, true) == 0);
-#endif
-
```

Comment:
This is an important diagnostic tool, it prevents us from ever creating an imagine that isn't valid. I don't understand from your comment exactly why verify is failing, but maybe we should fix verify?

Reply:
I had a discussion about this with Alex C yesterday. The verification code is written to verify the content in disk as it is. However, we now have logic to change the time pairs read from the cell, which causes the verification to fail. Alex preferred not changing the verification to use the same logic as it may mask some problems in the future. I also think we should fix it but we should do that in a separate ticket. Here I temporarily fixed it by clearing the time pairs.

Reply:
I'm still confused about this: As far as I can tell, there’s a path where we build an image that won’t verify. But, we are asserting that those images can never be written to disk, and I don't understand how we can know that.

As per our conversation on Slack, I think we need to understand this better before committing this change.

Reply:
The problem is we write the stop timestamp as WT_TS_MAX for non-timestamped delete and change it to 1 when we read it back to memory. We should directly write 1 to cell instead of  WT_TS_MAX in this case.